pkg/collector/corechecks/containers/cri/cri.go

```diff
@@ -108,22 +108,25 @@ func (c *CRICheck) generateMetrics(sender aggregator.Sender, containerStats map[
 			continue
 		}
 
+		ctnStatus, err := criUtil.GetContainerStatus(cid)
+		if err != nil {
+			log.Errorf("Could not retrieve the status of container %q: %s", cid, err)
```

Comment:
This can be very verbose as we run the check continuously, let's lower to `Debug` wdyt?